### ISSUE TITLE
Do not add trailing zeroes for numeric values in table-generator

### ISSUE THREAD

**Issue by PhilippWendler** [sosy-lab/benchexec#272]
If a user specifies for example three significant digits for a table column, but the value in the input file has only two digits (e.g., `1.1`), table-generator should not output `1.10` but `1.1` to not give a wrong indication of the precision of the value.

**Comment by PhilippWendler** [comment]
I wrote some unit tests for this, and I am surprised to see that this seems to be almost implemented already. @leostrakosch Is this correct and can you confirm that this is implemented? Did I overlook anything in the tests, or are there other interesting tests for this functionality that should be added? If yes, could you please add them?

It seems that the only incorrectly formatted number is the value zero, for which trailing zeros get added, probably as a result of #243. Given that the rest of this issue is already implemented, I guess doing it for zero should be easy. Could you please do this and adjust the unit tests that are marked with `FIXME` accordingly?

**Comment by leostrakosch** [comment]
I intended to implement it without adding additional zeros, so if it isn't working, it's an "unintended" bug.

> It seems that the only incorrectly formatted number is the value zero, for which trailing zeros get added, probably as a result of #243

I did it on purpose for zeros with the changes in #243 because I thought that that's what we wanted.
So just to reiterate and to be on the safe side:
`no. of zeros after decimal = min(no. of zeros after decimal in input value, max no. of decimal places in column)`

Right?
